libmamba/src/core/satisfiability_error.cpp

```diff
@@ -1144,13 +1144,13 @@ namespace mamba
                 m_pbs.graph().number_of_edges() + m_pbs.graph().number_of_nodes()
             );
             auto [out, _] = visit_node(m_pbs.root_node(), path.begin());
-            path.resize(out - path.begin());
+            path.resize(static_cast<std::size_t>(out - path.begin()));
             return path;
         }
 
         auto TreeDFS::node_uninstallable(node_id id) -> Status
         {
-            auto installables_contains = [&](auto&& id) { return leaf_installables.contains(id); };
+            auto installables_contains = [&](auto&& lid) { return leaf_installables.contains(lid); };
```

Comment:
`lid` feels weird here. Perhaps using `the_id`?

Reply:
lid stands for `local id`, this is king of a convention like `m_` and `p_` for data member.